acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(
+        self,
+        exc_type: type[BaseException] | None,
+        exc_val: BaseException | None,
+        exc_tb: Any | None,
+    ) -> None:
+        await self.close()
+
+    async def close(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `close`

Add return type annotation: `None`

(ANN201)